Sources/Commands/SwiftTool.swift

```diff
@@ -624,14 +633,15 @@ public class SwiftTool {
         return buildOp
     }
 
-    func createBuildSystem(explicitProduct: String? = nil, useBuildManifestCaching: Bool = true, buildParameters: BuildParameters? = nil) throws -> BuildSystem {
+    func createBuildSystem(explicitProduct: String? = nil, buildParameters: BuildParameters? = nil) throws -> BuildSystem {
+        print("createBuildSystem \(try self.canUseCachedBuildManifest())")
```

Comment:
@tomerd it looks like some sort of leftover debugging message, is it?

Reply:
good catch thanks

Reply:
https://github.com/apple/swift-package-manager/pull/3167